components/validator/introduction.rst

```diff
@@ -0,0 +1,74 @@
+.. index::
+   single: Validator
+   single: Components; Validator
+
+The Validator Component
+=======================
+
+    The Validator component provides tools to validate values following the
+    `JSR-303 Bean Validation specification`_.
+
+Installation
+------------
+
+You can install the component in 2 different ways:
+
+* :doc:`Install it via Composer </components/using_components>` (``symfony/validator`` on `Packagist`_);
+* Use the official Git repository (https://github.com/symfony/Validator).
+
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+The Validator component behavior is based on two concepts:
+
+* Contraints, which define the rules to be validated;
+* Validators, which are the classes that contain the actual validation logic.
+
+The following example shows how to validate that a string is at least 10
+characters long::
+
+    use Symfony\Component\Validator\Validation;
+    use Symfony\Component\Validator\Constraints\Length;
+
+    $validator = Validation::createValidator();
+    $violations = $validator->validateValue('Bernhard', new Length(array('min' => 10)));
```

Comment:
note this call is deprecated since ``Symfony 2.5``, also we can pass an array of constraints too:

```php
use Symfony\Component\Validator\Validation;
use Symfony\Component\Validator\Constraints\Length;
use Symfony\Component\Validator\Constraints\NotBlank;

$validator = Validation::createValidator();
$violations = $validator->validate('Bernhard', array(
    new Length(array('min' => 10)),
    new NotBlank()
));
```